lib/base/error-handler.js

```diff
@@ -34,7 +34,7 @@
         "INVALID_TEMPLATE" : "Invalid template name",
         "INVALID_FILE" : "Invalid file",
         "INVALID_VALUE" : "Invalid value",
-        "INVALID_DEVICENAME" : "Invalid device name. Do not use letters starting with '%' or '$'",
+        "INVALID_DEVICENAME" : "Invalid device name. Please use letters, numbers and allowed characters ('-', '_', '#')",
```

Comment:
allowed special characters('-', '_', '#')

special and no space between characters with (

Reply:
Done.